### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by mattiaerre**
# Description

- set `run_spec` as global (eslint)

- fix package.json [WIP]

- remove 'use strict'

### CODE REVIEW

**Comment by mattiaerre** [comment]
In addition, it'd be nice if:

1. @federicobond and @j-f1 you could fix your info as contributors in the package.json file

2. see if `raw-serializer.js` and `run_spec.js` could be consumed as modules; not sure if they are already available but it'd be cool to extract them from prettier; what do you think?

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
```
Comment (j-f1) [inline .eslintrc]: Can you add ESLint to the project explicitly (i.e. so `npm run lint` will run it?

Comment (mattiaerre) [reply .eslintrc]: oh snap! 😊 I'll do it!

File: package.json [hunk 2]
```diff
@@ -5,13 +5,14 @@
   "main": "src",
   "scripts": {
     "prettier": "prettier \"{.,{src,tests_config}/**}/*.{js{,on},md}\" \"tests/**/jsfmt.spec.js\"",
+    "eslint": "eslint src/**",
```
Comment (mattiaerre) [inline package.json]: I would keep all the scripts @j-f1 what do you think?

Comment (j-f1) [reply package.json]: Yeah let’s keep these, but can you add `npm run eslint` to the `lint` script and `npm run eslint -- --fix` to the `lint:fix` script?

**Comment by mattiaerre** [comment]
hi @federicobond I've added your email and if the CI is 💚 I'm going to merge this PR; let me know if you want me to use a different email; also re your print function I'll be more than happy to integrate it myself using your gist if you like and you are busy; let me know what you prefer. thanks so much,

**Comment by j-f1** [comment]
> @j-f1 shall we merge?

SGTM 👍

**Pull request merged**